kubernetes/validators.go

```diff
@@ -211,3 +210,8 @@ func validateAttributeValueIsIn(validValues []string) schema.SchemaValidateFunc
 
 	}
 }
+
+func validateStatefulSetUpdateStrategy(value interface{}, key string) (ws []string, es []error) {
```

Comment:
I don't see anything using this. Is it still needed?

Reply:
No, this is isn't used yet. It's a stub for a future implementation, as I need to research how to implement validators that must account for values elsewhere in the state.

I'll remove the stub until it has a meaningful implementation.